src/components/Actions.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
избавляйся от записей `React.`
```suggestion
import React, { FC } from 'react';
```